lib/sedna/service/server/aggregation.py

```diff
@@ -266,3 +269,55 @@ async def client_info(self, request: Request):
         if client_id:
             return server.get_client(client_id)
         return WSClientInfoList(clients=server.client_list)
+
+
+class AggregationServer():
+    def __init__(self, data=None, estimator=None,
+                 aggregation=None, transmitter=None,
+                 chooser=None) -> None:
+        # set parameters
+        server = Config.server._asdict()
+        clients = Config.clients._asdict()
+        datastore = Config.data._asdict()
+        train = Config.trainer._asdict()
+
+        if data is not None:
+            for xkey in data.parameters:
+                datastore[xkey] = data.parameters[xkey]
+            Config.data = Config.namedtuple_from_dict(datastore)
+
+        self.model = None
+        if estimator is not None:
+            self.model = estimator.model
+            for xkey in estimator.hyperparameters:
+                train[xkey] = estimator.hyperparameters[xkey]
+            Config.trainer = Config.namedtuple_from_dict(train)
+
+        server["address"] = "0.0.0.0"
+        server["port"] = 7363
```

Comment:
magic number

Reply:
Use the environment variable `AGG_BIND_PORT` instead, see https://github.com/kubeedge/sedna/blob/df6c88ba26362eefbefbde97b8706a6b52f40f7a/pkg/globalmanager/controllers/federatedlearning/federatedlearningjob.go#L428-L435

Reply:
updated